index.html

```diff
@@ -1,47 +1,77 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-	<link rel="stylesheet" type="text/css" href="css/styles.css">
-	<meta charset="UTF-8">
-	<meta name="viewport" content="width=device-width, initial-scale=1.0">
-	<title>Tribute to Steve Jobs</title>
-</head>
-<body>
+  <head>
+    <link rel="stylesheet" type="text/css" href="css/styles.css" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Tribute to PELE</title>
+  </head>
+  <body>
     <main id="main">
       <div id="heading" class="center">
         <h1 id="title" class="center">Pele</h1>
         <p class="font"><i>The greatest football player of all time</i></p>
       </div>
       <div id="img-div">
-        <img class="center" id="image" src="https://cdn.britannica.com/38/211838-050-3D3D2391/Pele-Brazilian-soccer-player-athlete-circa-1970s.jpg" alt="Pele">
+        <img
+          class="center"
+          id="image"
+          src="https://cdn.britannica.com/38/211838-050-3D3D2391/Pele-Brazilian-soccer-player-athlete-circa-1970s.jpg"
+          alt="Pele"
+        />
         <div id="img-caption" class="font">Pele in action</div>
       </div>
       <div id="tribute-info" class="center">
-    <h2 class="font">Here's a time-line of Pele's life:</h2>
-    <ul class="font" style="text-align: justify;">
```

Comment:
Please avoid using inline CSS styling and put all your CSS codes in your CSS File, this way you or anyone working on your code can maintain your code easily and your code will be clean and readable.